packages/tokens/playwright.config.ts

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
this rule is missleading, nx dep graph properly adds playwright to project dependencies